nemo/utils/exp_manager.py

```diff
@@ -281,17 +287,25 @@ def exp_manager(trainer: 'pytorch_lightning.Trainer', cfg: Optional[Union[DictCo
     logging.info(f'Experiments will be logged at {log_dir}')
     trainer._default_root_dir = log_dir
 
+    if cfg.log_local_rank_0_only and cfg.log_global_rank_0_only:
+        raise ValueError(
+            f"Cannot set both log_local_rank_0_only and log_global_rank_0_only to True. Please set either one or neither."
```

Comment:
Why can't both be set - log global rank 0 and all local rank 0s (duplicate global 0 and local 0 if that's easier) then?

Reply:
But thats the same behaviour as just log_local_rank_0_only no?

Reply:
lets take an example, if using 4 nodes with 16 GPUs.

When setting `log_local_rank_0_only`, you'll end up with 4 log files `nemo_log_globalrank-0_localrank-0.txt`, `nemo_log_globalrank-16_localrank-0.txt`, `nemo_log_globalrank-32_localrank-0.txt`, `nemo_log_globalrank-48_localrank-0.txt`

With `log_global_rank_0_only`, you'll get just one file `nemo_log_globalrank-0_localrank-0.txt`, which is already a part of `log_local_rank_0_only`.

Reply:
Ok so local 0 global 0 anyway is global 0, that's fine to make them exclusive flags then.